### PR TITLE
docker: test image builds with Travis CI

### PR DESCRIPTION
**Pull request by abitrolly**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -132,3 +132,29 @@ jobs:
         - rm -rf snapcraft
       script:
         - SNAPCRAFT_PACKAGE_TYPE=snap ./runtests.sh tests/integration/store
+
+    - stage: docker
+      env:
+        RISK: "stable"
```
[the remark this answers is not in the document]
Comment (abitrolly) [reply .travis.yml]: Let's try. Although I would prefer https://github.com/snapcore/snapcraft/pull/2673 to be merged to remove duplication.

[the remark this answers is not in the document]
Comment (abitrolly) [reply .travis.yml]: @sergiusens matrix expansion didn't work for stages - https://travis-ci.org/snapcore/snapcraft/builds/639097783

[the remark this answers is not in the document]
Comment (abitrolly) [reply .travis.yml]: Here is the issue https://github.com/travis-ci/travis-ci/issues/8295

[the remark this answers is not in the document]
Comment (sergiusens) [reply .travis.yml]: oh well, thanks for giving it a go.

**Pull request merged**